### ISSUE TITLE
Observaciones 

### ISSUE THREAD

**Issue by noedavico** [Antoniomorales17/Contact-List-React#2]
- No hay validación de formularios implementada en el código. Sería recomendable agregar validaciones para garantizar que los campos obligatorios estén completos y que los datos ingresados ​​sean válidos. Por ejemplo en el campo telefono puedo ingresar letras. 
- Estilos los icono de direccion y telefono estan invertidos :)
- Cuando se agrega un contacto y se da al boton save si esta ok deberia redirigir a la lista de contacto o dar un mensaje contacto agragado con exito.
- En la función setStore dentro de getState, en lugar de usar Object.assign para fusionar los cambios en el estado, puedes utilizar el Spread Operator( ...).

**Comment by agarzon48** [comment]
Estoy de acuerdo con Noe. Voy a rechazar la entrega para que puedas implementar esas mejoras, avisa por este hilo cuando estén!

Otras cosillas menores que he visto:

* En Layout tienes un par de divs que no hacen nada. Mejor si los quitas, nos ahorramos dos nodos y eso mejora la performance de nuestra app. Lo mismo en componentes como Contact o AddContact. En general, si el div ni siquiera tiene una clase, sobra.
* En contacts estás utilizando el index de cada contacto como key para su componente Contact. Es mejor que utilices el user.id, porque es único y nunca va a cambiar. En cambio, basta con hacer list.unshif('asdf') para que todos los índices de todos los contactos cambien y se tengan que renderizar de nuevo. Esto también mejora la performance de tu app

**Comment by Antoniomorales17** [comment]
- He aplicado las mejoras y cambios propuestos a la app.
- Aún así ahora me esta dando error con la solicitud "Delete" y no me deja eliminar a los usuarios. Seguiré probando por si fuera culpa de Api.
Dadme feed para seguir
Gracias
